servers/client-api/config/supergraph.yaml

```diff
@@ -1,10 +1,76 @@
-federation_version: =2.4.7
+federation_version: =2.7.0
 subgraphs:
-  products:
+  # Pocket Monorepo
+  annotations-api:
+    routing_url: https://annotations-api.readitlater.com
+  #   schema:
+  #     subgraph_url: https://annotations-api.readitlater.com
+    schema:  # Schema downloaded from GraphOS registry, does not poll for updates
+      graphref: pocket-client-api@current
+      subgraph: annotations-api
+  featureflags:
+    routing_url: https://featureflags.readitlater.com/graphql
+    schema:
+      subgraph_url: https://featureflags.readitlater.com/graphql
+  image-api:
+    routing_url: https://image-api.readitlater.com
+    schema:
+      subgraph_url: https://image-api.readitlater.com
+  list-api:
+    routing_url: https://list-api.readitlater.com
+    schema:
+      subgraph_url: https://list-api.readitlater.com
+  parser:
+    #routing_url: https://parser-graphql-wrapper.readitlater.com
```

Comment:
reason this one is commented out?

Reply:
Was running it locally against my local parser